rllib/policy/torch_policy_v2.py

```diff
@@ -1211,7 +1211,11 @@ def _compute_action_helper(
                 # ACTION_DIST_INPUTS field returned by `forward_exploration()` ->
                 # Create a distribution object.
                 action_dist = None
-                if SampleBatch.ACTION_DIST_INPUTS in fwd_out:
+                # Eventually the RLModule has already computed actions.
+                if (
+                    SampleBatch.ACTION_DIST_INPUTS in fwd_out
+                    and SampleBatch.ACTION_LOGP not in fwd_out
```

Comment:
Can you explain this change? I don't follow :)

Just thinking from a very top-level: Shouldn't this PR not be concerned about policies at all anymore?